pages/windows/taskkill.md

```diff
@@ -0,0 +1,27 @@
+# taskkill
+
+> Terminate a process by its process id or name.
+
+- Terminate a process by its id:
+
+`taskkill /pid {{process_id}}`
+
+- Terminate a process by its name:
+
+`taskkill /im {{process_name}}`
+
+- Forcefully terminate a specified process (must be combined with a process id or name):
+
+`taskkill /f`
+
+- Terminate a process and its child processes (must be combined with a process id or name):
+
+`taskkill /t`
+
+- Terminate a process on a remote machine:
+
+`taskkill /s {{remote_name}}`
```

Comment:
Shouldn't we need to give the process id or name here too ?

Reply:
Yes, it can be combined with any of the above commands, so I wasn't sure if it needed to mention it.

Reply:
Dunno, looks confusing to me. Mentioning it thrice is also redundant. Best way is to go with @sbrl's original recommendation. Use `/pid` for `/f`, `/im` for `/t`, and again `/pid` for `/s`. That wil clearly indicate that pid or process_name can be mixed. And remove the text in `()`.

Reply:
Sounds good to me. :+1: I'll make that change in a bit.